packages/react/src/lib/renderComponent.tsx

```diff
@@ -40,6 +42,8 @@ export interface RenderResultConfig<P> {
   styles: ComponentSlotStylesPrepared
   accessibility: AccessibilityBehavior
   rtl: boolean
+  renderer: Renderer
```

Comment:
actually, we shouldn't pass `renderer` to component's `render` method - could you, please, suggest where this is necessary thing to do (sorry if I am missing something)? Also, the same most probably true for `disableAnimations` flag - we need to pass only those bits that are necessary for the component to finish rendering: classes (styles), ElementType, unhandledProps ...

Reply:
For the `renderer`, currently we are using it to render the keyframes in the animations. If we use the https://github.com/rofrischmann/fela/tree/master/packages/fela-plugin-embedded - @layershifter has confirmed that the animations are working as expected in rtl now with this plugin, then we can skip the `renderer` as well. For the `disableAnimations`, not sure if we should skip it, as for some components, if the animation are disabled they may want to show different slots, or style the component differently (imagine Loader with `disaledAnimations` - they may want to provide some circle etc)